modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -112,8 +122,48 @@ command and can be used to further customize the mirroring configuration.
 . If you used the `--manifests-only` flag in the previous step and want to mirror
 only a subset of the content:
 
-.. Modify the list of images in your `mapping.txt` file to your
-specifications.
+.. Modify the list of images in your `mapping.txt` file to your specifications. If
+you are unsure of the exact names and versions of the subset of images you want
+to mirror:
+
+... Run the `sqlite3` tool against the temporary database that was generated by the
+`oc adm catalog mirror` command to retrieve a list of images matching a general
+search query.
++
+For example, to retrieve a list of images that that are similar to the string
+`clusterlogging.4.3`:
++
+[source,terminal]
+----
+$ echo "select * from related_image \
+    where operatorbundle_name like 'clusterlogging.4.3.%';" \
+    | sqlite3 -line /tmp/190214037/bundles.db <1>
+----
+<1> Refer to the previous output of the `oc adm catalog mirror` command to find the
+path of the database file.
++
+.Example output
+[source,terminal]
+----
+image = registry.redhat.io/openshift4/ose-logging-kibana5@sha256:aa4a8b2a00836d0e28aa6497ad90a3c116f135f382d8211e3c55f34fb36dfe61
+operatorbundle_name = clusterlogging.4.3.33-202008111029.p0
+
+image = registry.redhat.io/openshift4/ose-oauth-proxy@sha256:6b4db07f6e6c962fc96473d86c44532c93b146bbefe311d0c348117bf759c506
+operatorbundle_name = clusterlogging.4.3.33-202008111029.p0
+[...]
+----
+
+... Use the results from the previous step to edit the `mapping.txt` file to only
+include the subset of images you want to mirror. For example, you can conclude
+from the previous example output that the following lines exist in your
+`mapping.txt` file to match the `image` entries from the database:
++
+.Matching image mappings in `mapping.txt`
+[source,terminal]
```

Comment:
Not sure that this should be source,terminal if this is the contents of a .txt file

Reply:
Going with `[source,txt]`.